scene/2d/room_2d.h

```diff
@@ -0,0 +1,25 @@
+#ifndef ROOM_2D_H
+#define ROOM_2D_H
+
+#include "scene/2d/node_2d.h"
+#include "scene/resources/space_2d.h"
+
+class Room2D : public Node2D {
+
+	GDCLASS(Room2D, Node2D)
+
+private:
+	Ref<Space2D> space_2d;
+
+protected:
+	static void _bind_methods();
+
+public:
+	void set_space_2d(const Ref<Space2D> &p_space_2d);
+	Ref<Space2D> get_space_2d() const;
+
+	Room2D();
+	~Room2D();
+};
+
+#endif // ROOM_2D_H
```

Comment:
Add a newline at end of file, this is for proper POSIX compliance.

Reply:
This is fixed now